package.json

```
@@ -6,6 +6,9 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "bootstrap": "^4.6.0",
+    "jquery": "^3.6.0",
```

`3.6.0` はハンバーガーメニューにエラーが出ていますので，3.5.1にしてください。

```suggestion
    "jquery": "^3.5.1",
```

その後次を実行してからプッシュしてください。

```
yarn install --check-files
```

---

お疲れ様です！
修正致しましたので、確認お願い致します。